### ISSUE TITLE
Device does not change parameters/changes leads to same value

### ISSUE THREAD

**Issue by TutenStain** [AlCalzone/node-zwave-js#5139]
I have a couple of Everspring AC301 TRVs and since a while (cannot pinpoint when, could be months) they stopped responding to parameter changes. Example: I change `Thermostat Mode v3 [84-64-0-mode] Thermostat mode` from heat to energy heat but the setting is never updated on the device. Going by the logs it seems to attempt to change to the same value it currently has? 

The log:

`2022-10-01 15:01:00.871 INFO Z-WAVE: Node 84: value updated: 64-0-mode 1 => 1`

```
 2022-10-01T13:01:00.874Z - VALUE UPDATED 
Arg 0:
└─commandClassName: Thermostat Mode
└─commandClass: 64
└─property: mode
└─endpoint: 0
└─newValue: 1
└─prevValue: 1
└─propertyName: mode
```

The full debug info log is attached if needed.

[log.txt](https://github.com/zwave-js/zwave-js-ui/files/9690373/log.txt)

**Comment by robertsLando** [comment]
Did you tried to re-interview your device first?

**Comment by TutenStain** [comment]
> Did you tried to re-interview your device first?

Yes, I have both re-interviewed the device and have gone through the process of “replacing” it in the UI. All of my many of the same devices exhibits the same behavior. All have worked previously.

**Comment by robertsLando** [comment]
Please attach zwave-js [driver logs](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=generating-log-files)

**Comment by TutenStain** [comment]
Attaching logs. Should be the last thing in the files as I enabled logging, attempted to change value and then downloaded the logs.

[zwave-js-ui-store.zip](https://github.com/zwave-js/node-zwave-js/files/9697604/zwave-js-ui-store.zip)

**Comment by robertsLando** [comment]
```
2022-10-03T12:45:21.387Z DRIVER « [Node 084] [REQ] [ApplicationCommand]
                                  └─[SupervisionCCReport]
                                      session id:          42
                                      more updates follow: false
                                      status:              Fail
                                      duration:            0s
2022-10-03T12:45:21.392Z CNTRLR   [Node 084] [setValue] result of SET_VALUE API call for ThermostatModeCCAPI: (S
                                  upervisionResult)
                                    status:   Fail
2022-10-03T12:45:21.393Z CNTRLR   [Node 084] [setValue] not updating value
```

@AlCalzone I think the problem could be because the node is reporting to support SpervisionCC but it actually doesn't support it, may need a compat flag too work?

**Comment by kpine** [comment]
See release https://github.com/zwave-js/node-zwave-js/releases/tag/v10.3.0.

> Disable Supervision for Everspring AC301 (https://github.com/zwave-js/node-zwave-js/pull/5119)

**Comment by robertsLando** [comment]
Ok so just wait for https://github.com/zwave-js/zwave-js-ui/pull/2702 to be merged

**Comment by TutenStain** [comment]
Awesome, thanks!

**Comment by robertsLando** [comment]
Merged and released (8.1.0)
